integration_tests/iceberg-cdc/run_test.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
This is not in ci? See https://github.com/risingwavelabs/risingwave/blob/61a5bd59a31992f0c514150913e7d41c32bee246/ci/workflows/integration-tests.yml#L17

Reply:
I have add it, do we have a way to test it in this PR?

Reply:
No, it's tested in daily ci. We can wait an see.